keyboards/kingly_keys/ropro/rules.mk

```diff
@@ -0,0 +1,67 @@
+# MCU name
+MCU = atmega32u4
```

Comment:
As above, you don't need the stuff between `MCU` and `BOOTLOADER`.

Reply:
There is a physical button. But that was my attempt at lowing the Firmware size.  It kept maxing out past the size limit.

Reply:
Ah, okay.

Well, a simple solution for that may be to set `LINK_TIME_OPTIMIZATION_ENABLE = yes` in your `rules.mk` file.  This will enable ... well, link time optimization, which may greatly reduce the firmware size.  It also turns off action macros and action function, since they error out with LTO enabled (but this doesn't affect the `process_record_*` macros, at all.